packages/11ty/_includes/components/figure/image.js

```diff
@@ -10,21 +10,34 @@ const path = require('path')
  * @return     {String}  An HTML <img> element
  */
 module.exports = function(eleventyConfig, globalData) {
+  const canvasPanel = eleventyConfig.getFilter('canvasPanel')
   const figurecaption = eleventyConfig.getFilter('figurecaption')
   const figurelabel = eleventyConfig.getFilter('figurelabel')
   const figuremodallink = eleventyConfig.getFilter('figuremodallink')
   const markdownify = eleventyConfig.getFilter('markdownify')
 
   const { imageDir, figureLabelLocation } = globalData.config.params
 
-  return function({ alt='', caption, credit, id, label, src='' }) {
-
+  return async function({ alt='', canvasId, caption, choices, credit, id, iiifContent, label, manifestId, preset, src='' }) {
     const imageSrc = path.join(imageDir, src)
     const labelElement = figurelabel({ caption, id, label })
-
-    let imageElement = `
-      <img alt="${alt}" class="q-figure__image" src="${imageSrc}" />
-    `
+    const srcParts = src.split(path.sep)
+    const hasTiles = srcParts[srcParts.length - 1] === 'tiles'
+    const hasManifestAndCanvasIds = (!!canvasId && !!manifestId) || !!iiifContent || !!choices
+
+    let imageElement;
+
+    switch (true) {
+      case hasManifestAndCanvasIds:
```

Comment:
Since this is also `true` when there is `iiifConent` or `choices` perhaps this should be named `hasCanvasPanelProps` or `useCanvasPanel`
```suggestion
    const hasCanvasPanelProps = (!!canvasId && !!manifestId) || !!iiifContent || !!choices

    let imageElement;

    switch (true) {
      case hasCanvasPanelProps:
```